pkg/envtest/server.go

```diff
@@ -299,7 +299,12 @@ func (te *Environment) Start() (*rest.Config, error) {
 	te.CRDs = crds
 
 	log.V(1).Info("installing webhooks")
-	if err := te.WebhookInstallOptions.Install(te.Config); err != nil {
+	if te.WebhookInstallOptions.CustomTLSConfig != nil {
+		err = te.WebhookInstallOptions.InstallWithCustomTLS(te.Config)
+	} else {
+		err = te.WebhookInstallOptions.Install(te.Config)
+	}
```

Comment:
Do we need a separate `InstallWithCustomTLS` method? Or could we just make `Install` know how to handle the `CustomTLSConfig` field?